paasta_tools/cli/cmds/spark_run.py

```diff
@@ -111,6 +134,13 @@ def add_subparser(subparsers):
         help="Use the provided image to start the Spark driver and executors.",
     )
 
+    list_parser.add_argument(
+        "-e",
+        "--enable-k8s-autogen",
+        help="Auto generate pod template with weighted pod affinity used in executors.",
```

Comment:
Maybe change this to something more relevant in the language which spark-users are familiar with.
Something like, "Enabling/Disabling this flag would enable round-robin. That means, executors would be spread out across nodes. By default, executors are preferred to be launched on the same node."?